uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java

```diff
@@ -141,6 +141,11 @@ private void signIn(@NonNull final String username,
         new AadPromptHandler(promptHandlerParameters).handlePrompt(username, password);
     }
 
+    public String handleSignInWithoutPrompt() {
```

Comment:
your method name says sign in without prompt. But you are returning a secret. These look a bit disconnected. I would suggest you move your assert statements to this method. Assert here itself if the token is valid. then you do not have to return the token to the test